### ISSUE TITLE
Feature request: syntax highlight in Markdown

### ISSUE THREAD

**Issue by raingloom** [keybase/client#8849]
# Allow adding a language to code blocks, same way Github does it

## how it is now

    ```c++
    int main(){}
    ```
simply becomes:

```
int main(){}
```

## how it should be

    ```c++
    int main(){}
    ````
should render to

```c++
int main(){}
```

Discord does this too and it's really nice.

**Comment by konsumer** [comment]
[prism](http://prismjs.com/) could make this fairly painless.

**Comment by jbsmith7741** [comment]
code blocks would be really nice to have.

**Comment by zJeremiah** [comment]
Since this is used by many devs, sharing code with syntax highlighting would be very helpful.

**Comment by nall** [comment]
Would be great to have this.

**Comment by moltar** [comment]
This would be a KILLER feature over Slack, which does not support it after all these years 🙄

**Comment by chrispruitt** [comment]
This would be great to have!

**Comment by l3uddz** [comment]
Agreed, this is probably one of the only things I miss from Discord (although I wont miss that rubbish snippet size limit discord impose).

**Comment by T99** [comment]
Has this moved forward at all or been considered by the team? Slack offers full support for this now, and this is the main thing that I miss from that and Discord.

Would love to see this in Keybase.

**Comment by opr** [comment]
Hi I have started work on this, please view my progress here: https://github.com/opr/client/commit/e8db611521cccb65e14161016e96c98f60a61f15

as mentioned in the commit, I need help with how to add the styles from highlight.js. At the moment I just added it to react.tsx but this is not the right place (I didn't even know where to begin to look at adding third party stylesheets). If you show me how to add this properly I will do it!

Also I need to know if this approach is going to work on the React Native app? (since it uses divs, will this work?) comments would be very much appreciated.
